src/app/index.tsx

```diff
@@ -86,6 +87,11 @@ export default (options: Options) => {
     ...hooks.map((hook) => hook(services)),
   ];
 
+  /* istanbul ignore next */
+  if (process.env.REACT_APP_ENV === 'development') {
```

Comment:
Why not include it in production? Our code is public & it could help debug errors that occur in production that are sent to sentry.

Reply:
i didn't include it in production because it adds an async middleware step to every action dispatch where it resolves the sourcemaps for the trace before allowing the action to continue, it seemed probably bad to do that.

we can definitely use the same library to decorate any messages or errors that we send to sentry when we make that service though.